src/qtism/data/content/interactions/ExtendedTextInteraction.php

```diff
@@ -473,7 +475,7 @@ public function hasExpectedLines(): bool
     /**
      * Set the format of the text entered by the candidate.
      *
-     * @param int $format A value from the TextFormat enumeration.
+     * @param  int $format A value from the TextFormat enumeration.
```

Comment:
I don't think we need to align the docblock elements like that.
I'd rather have all of them consistent and with a single space as a delimiter.